shell/platform/linux/fl_renderer.cc

```diff
@@ -99,7 +168,21 @@ gboolean fl_renderer_make_current(FlRenderer* self, GError** error) {
   if (!eglMakeCurrent(priv->egl_display, priv->egl_surface, priv->egl_surface,
                       priv->egl_context)) {
     g_set_error(error, fl_renderer_error_quark(), FL_RENDERER_ERROR_FAILED,
-                "Failed to make EGL context current");
+                "Failed to make EGL context current: %s", get_egl_error());
+    return FALSE;
+  }
+
+  return TRUE;
+}
+
+gboolean fl_renderer_make_resource_current(FlRenderer* self, GError** error) {
+  FlRendererPrivate* priv =
+      static_cast<FlRendererPrivate*>(fl_renderer_get_instance_private(self));
```

Comment:
And then early return here if it's null.

Reply:
Not sure what this comment is referring to?

Reply:
If `resource_surface` wasn't successfully created, none of the code below should be executed.

Reply:
Ah, GitHub is showing this on a different line but I know what you're referring to.